### DISCUSSION TITLE
Airflow Run with Config Example

### DISCUSSION THREAD

**Discussion by shwar123** [apache/airflow#29345] [Q&A]
Dear Team,

I have a requirement like i need to print HELLO WORLD ESHWAR Text  from airflow , in run time only i have to pass value ESHWAR. it should not be hardcoded  inside the text it should come in the fly

Can anyone help on this please, 

Thank YouEshwar

**Comment by potiuk** [comment]
https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/params.html#params

**Reply by shwar123** [reply]
Thanks!
Let me Explain my issue clearly, An sample example 

when ever i wanted to run my Airflow dag I have to pass Getdate ()  then only Airflow Dag should RUN. it has to ask in the run time once clicked on Trigger till that parameter received from manually it should not allow to Run

Please help me with some same code

Thanks
Eshwar

**Reply by potiuk** [reply]
I have completely no clue what you are asking about.

Please read the docs and go through examples - airflow has plenty of them. And whatever you want to do to it is likely basic feature of airflow that has examples ready when you install airflow and keep examples on.

I suggest you to look at those first (and clarity in asking questions might go a long way if you ask for help).

**Reply by shwar123** [reply]
Inconvenience regretted!!

I am sorry actually I am pretty new to Airflow!

My scenario is 

I have a dag created with simple to print 'Hello world' with python and it is working fine and after performed run with trigger dag ran and given Hello World it is absolutely fine.

But as per my requirement I have to run the above dag based on a date . For example if I wanted to run today I have to pass a dynamic parameter getdate or today date like Yyyy/mm/dd then only it has to trigger and give hello world. While triggering the dag should ask enter value for date of i enter date it should trigger.

Hoping I have explained clearly, kindly do the needful

Thank you 

Thank you!

**Reply by potiuk** [reply]
Read the docs please and see if it fits your needs. I still do not understand if it does - but this link 
 explains how you can add parameter when you trigger the DAG https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dag-run.html#external-triggers (And the previous one shows how you can validate it). Both contain examples and screenshots so follow them and explore.
